docs/deployment/netconf/registries.mdx

```diff
@@ -8,20 +8,21 @@ The wasmCloud host supports pulling images for [components](/docs/concepts/compo
 
 ## Allowing Insecure Access to a Registry
 
-By default wasmCloud will pull all images from a registry over HTTPS. If you need to pull images from a registry that does not have a valid certificate, you can allow wasmCloud to pull images from that registry by setting the `WASMCLOUD_OCI_ALLOWED_INSECURE` environment variable. For example:
+By default wasmCloud will pull all images from a registry over HTTPS. If you need to pull images from a registry that does not have a valid certificate, you can allow wasmCloud to pull images from that registry by setting the `ALLOWED_INSECURE` environment variable. For example:
```

Comment:
were we going to change this? it's still `WASMCLOUD_OCI_ALLOWED_INSECURE` AFAICT